package HelloKittyMsgData;
import "trade.proto";

message RecordSerialize
{
	 repeated SellPaperCell advertise = 1;
}
